src/lib/familiar.ts

```diff
@@ -17,3 +25,32 @@ export function noncombatFamiliar(): Familiar {
     return $familiar`Disgeist`;
   }
 }
+
+export function freeRunsFamiliar(): Familiar {
+  function pickFamiliar(): Familiar {
```

Comment:
Let's keep the project style consistent; lets use a variable for this function instead of the function keyword. E.g
```
const pickFamiliar = () => { . . . }
```